Jenkinsfile

```
@@ -61,7 +58,7 @@ pipeline {
                                             sh 'make build'
                                             sh 'make test'
                                             // If the tests are passing for Linux AMD64, then we can build all the CPU architectures
-                                            sh 'docker buildx bake --file docker-bake.hcl linux'
+                                            sh 'make buildall'
```

```suggestion
                                            sh 'make build-all'
```

nit: use a dashed case as word separator to be homogeneous with the ci-init or check-reqs targets

---

We can't use `build-all` unfortunately as there is another make target `build-%` just above already catching every call to `make build-`*.

---

WDYT of `complete-build`?

---

Or "everything"? `make everything`

---

Or `every-build`?

---

After discussing it together, `every-build` seems the best, avoiding the issue of the "build-" prefix already taken and keeping the intent clear.